Mage.Sets/src/mage/cards/p/PrimalPlasma.java

```diff
@@ -119,8 +118,8 @@ public boolean replaceEvent(GameEvent event, Ability source, Game game) {
                         game.addEffect(new GainAbilitySourceEffect(DefenderAbility.getInstance(), Duration.Custom), source);
                         break;
                 }
-                permanent.getPower().modifyBaseValue(power);
-                permanent.getToughness().modifyBaseValue(toughness);
+                permanent.getPower().setModifiedBaseValue(power);
+                permanent.getToughness().setModifiedBaseValue(toughness);
```

Comment:
Whoops, I missed this. This card should be using the same approach as Primal Clay and Molten Sentry (this card is identical to Primal Clay except for being blue and not an artifact).

Reply:
what's wrong with this one?

Reply:
It should be creating a continuous effect to set the chosen P/T, rather than directly setting the values (which will be overwritten as soon as the game state is updated).

Reply:
Primal Clay (and two or three other cards based directly on it) are a special bizarre grandfathered-in exception to the layer system. The optional flying and defender abilities are *also* supposed to be copiable, and in xmage that only works with specific copy effects (e.g. Identity Thief) and completely by accident. I think we need to think of a new way to handle them properly.

Reply:
What you're suggesting is similar to how `TransformAbility` currently works. I've never really liked how TransformAbility is implemented as a continuous effect, and it requires some special cases to work with copying (so we'd have to add even more special cases to make copying Primal Clay work).

Reply:
I'm going to open a separate issue for this. I think that fixing them is beyond the scope of this PR